conf/views/viewAllArrivals.njk

```diff
@@ -21,6 +22,8 @@
         </a>
       </p>
 
+      {{ movementSearch( messages('movement.search.title'), mrn, "mrn", routes.controllers.arrival.ViewArrivalsSearchResultsController.onPageLoad("").url) }}
+
```

Comment:
We could make this string optional to avoid having to pass empty strings

Reply:
We could. I didn't because I didn't want it to really be optional. :) But that could be done.